Sources/SwiftParser/Lexer/LexemeSequence.swift

```diff
@@ -11,6 +11,7 @@
 //===----------------------------------------------------------------------===//
 
 @_spi(RawSyntax) import SwiftSyntax
+@_spi(BumpPtrAllocator) import SwiftSyntax
```

Comment:
I think we prefer to [pile up the attributes](https://github.com/apple/swift-syntax/blob/main/Sources/SwiftParser/TokenSpecSet.swift) on a single import.